src/EditorFeatures/Core/Host/IStreamingFindReferencesPresenter.cs

```diff
@@ -34,14 +34,14 @@ internal interface IStreamingFindUsagesPresenter
         /// items in isolation.</param>
         /// <param name="cancellationToken">External cancellation token controlling whether finding shoudl be canceled
         /// or not.  This will be combined with a cancellation token owned by the <see cref="FindUsagesContext"/>.
-        /// Callers should consider <see cref="FindUsagesContext.CancellationToken"/> to be the source of truth for
+        /// Callers should consider the cancellation token returned to be the source of truth for
         /// cancellation from that point on.</param>
```

Comment:
See comments elsewhere: this is a pretty dangerous pattern since it means callers may throw cancellation when the original caller didn't expect it. I wonder if that cancellation token confusion is partially the cause of mistakes here, and we aren't rectifying it. My gut is saying that starting a search isn't a cancellable operation  -- we're just opening up a dialog. The caller either never got to that point, or has chosen to show a dialog at which point the original cancellation is now useless -- there's no more waiting, and we're now only relying on some external cancellation UI.....right?

Reply:
resolved.